packages/matchbox/src/components/KeyboardKey/KeyboardKey.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { tokens } from '@sparkpost/design-tokens';
+
+const StyledKey = styled.kbd`
+  position: relative;
+  display: inline-block;
+  padding: 0 ${tokens.spacing_100};
+  height: ${tokens.sizing_450};
+
+  background: ${tokens.color_gray_100};
+  border-radius: 4px;
```

Comment:
Is this something we want to add to the tokens? Probably not necessary if we're just using 4px border radius in this one instance

Reply:
I thought about it, but I don't _think_ well be using it anywhere else